CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- Removed OpenSSL from all images.
```

Comment:
This is a breaking change so it should be tagged as such.

Reply:
Yeah, this will definitely be released as 0.2.0.

Reply:
By tagging I meant it should be mentioned in the ChangeLog. But that can be done before the release, too.